scipy/stats/kde.py

```diff
@@ -239,7 +239,11 @@ def evaluate(self, points):
                     self.d)
                 raise ValueError(msg)
 
-        result = zeros((m,), dtype=float)
+        if np.issubdtype(points.dtype, np.floating):
+            output_dtype = points.dtype
+        else:
+            output_dtype = np.float
```

Comment:
`np.float` is just an alias for the Python built-in `float`, and the use of that alias is discouraged.  I think most numpy devs would agree that exposing that name in the `numpy` namespace was a mistake.

Instead of `np.float`, use `np.float64`, which is the actual numpy data type that we want.

Reply:
thats good to know, I'd wondered at that before!